packages/video_player/video_player_avfoundation/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -678,4 +751,55 @@ - (void)setMixWithOthers:(FLTMixWithOthersMessage *)input
   }
 }
 
+- (nullable NSNumber *)isPictureInPictureSupported:
+    (FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  return @([AVPictureInPictureController isPictureInPictureSupported]);
+}
+
+- (void)setAutomaticallyStartsPictureInPicture:(FLTAutomaticallyStartsPictureInPictureMessage *)input
+                                        error:(FlutterError **)error {
+  FLTVideoPlayer *player = self.playersByTextureId[input.textureId];
+  [player
+      setAutomaticallyStartsPictureInPicture:input.enableStartPictureInPictureAutomaticallyFromInline
+                                                .boolValue];
+}
+
+- (void)setPictureInPictureOverlayRect:(FLTSetPictureInPictureOverlayRectMessage *)input
+                                 error:(FlutterError **)error {
+  FLTVideoPlayer *player = self.playersByTextureId[input.textureId];
+  [player setPictureInPictureOverlayRect:CGRectMake(input.rect.left.floatValue,
+                                                    input.rect.top.floatValue,
+                                                    input.rect.width.floatValue,
+                                                    input.rect.height.floatValue)];
+}
+
+- (BOOL)doesInfoPlistSupportPictureInPicture:(FlutterError **)error {
```

Comment:
This needs a declaration comment, per the ObjC style guide.

Reply:
I am not sure what I should do exactly here? I don't see any other comments on the other methods. Can you give me some guidance here?